generators/entity-client/templates/vue/src/test/javascript/e2e/entities/entity-page-object.ts.ejs

```diff
@@ -32,11 +32,11 @@ export default class <%= entityClass %>ComponentsPage {
   }
 
   async clickOnFirstDetailsButton() {
-    await this.detailButtons.first().click();
+    await this.detailButtons.last().click();
```

Comment:
mandatory, because of "faker" in dev

Reply:
If we keep this change, we should rename the method